travis-deploy.sh

```diff
@@ -1,26 +1,35 @@
 #!/usr/bin/env sh
 set -e
 
-export WEBDAV_USER="Bio-Formats"
-export UPDATE_SITE="Bio-Formats"
+export WEBDAV_USER=${WEBDAV_USER:-"Bio-Formats"}
+export UPDATE_SITE=${UPDATE_SITE:-"Bio-Formats"}
 
-# Define some variables 
-export IJ_PATH="$HOME/Fiji.app"
-export URL="http://sites.imagej.net/$UPDATE_SITE/"
-export IJ_LAUNCHER="$IJ_PATH/ImageJ-linux64"
-export PATH="$IJ_PATH:$PATH"
+# Define update site variables
+export URL=${URL:-"http://sites.imagej.net/$UPDATE_SITE/"}
+export LOGIN=${LOGIN:-"webdav:$WEBDAV_USER:$WIKI_UPLOAD_PASS"}
+export DIR=${DIR:-"."}
+
+# IJ Installation variables
+export IJ_PATH=${IJ_PATH:-"$HOME/Fiji.app"}
+export IJ_PLATFORM=${IJ_PLATFORM:-"linux64"}
+export IJ_LAUNCHER=${IJ_LAUNCHER:-"${IJ_PATH}/ImageJ-${IJ_PLATFORM}"}
+export IJ_ZIP=${ZIP:-"fiji-${IJ_PLATFORM}.zip"}
 
 # Install ImageJ
-mkdir -p $IJ_PATH/
-cd $HOME/
-wget --no-check-certificate https://downloads.imagej.net/fiji/latest/fiji-linux64.zip
-unzip fiji-linux64.zip
-$IJ_LAUNCHER --update update-force-pristine
+if [ ! -e "$IJ_PATH" ];
+then
+    mkdir -p "$IJ_PATH/"
+    pushd $HOME/
```

Comment:
Getting `./travis-deploy.sh: 22: ./travis-deploy.sh: pushd: not found` - see https://travis-ci.org/sbesson/bio-formats-fiji/builds/469604591#L1734